qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -105,7 +105,7 @@ def decompose_clifford_bm(clifford):
     ret_circ = QuantumCircuit(num_qubits, name=clifford_name)
     for qubit in range(num_qubits):
         pos = [qubit, qubit + num_qubits]
-        table = clifford.tableau()[pos][:, pos]
+        table = clifford.paulis.tableau()[pos][:, pos]
```

Comment:
There's still a bunch of these things doing method work within loops, but it's not _critical_ that we fix them right now - we're tight for the release, so we could fix them after.  If you've got time to do them now though, it'd be best.

Reply:
I believe I've fixed this and the others you pointed out. There may be others, but further improvements will be for another PR.